src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerGraph.java

```diff
@@ -59,6 +61,8 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.Stream;
 
+import static com.sun.org.apache.xml.internal.security.keys.keyresolver.KeyResolver.iterator;
```

Comment:
we're doing xml security now? :rofl:

Reply:
Oh crap, i guess the auto importer went bananas. ;)